include/WCSimDetectorConstruction.hh

```diff
@@ -71,6 +71,7 @@ public:
   G4String GetDetectorName()      {return WCDetectorName;}
   G4double GetWaterTubeLength()   {return WCLength;}
   G4double GetWaterTubePosition() {return WCPosition;}
+  G4double GetWaterTubeRadius()   {return WCRadius;}
```

Comment:
Thank you Jost, that works.
I removed the offensive line.

Federico

On 21/06/16 17:59, Jost Migenda wrote:
>
> In include/WCSimDetectorConstruction.hh 
> <https://github.com/WCSim/WCSim/pull/168#discussion_r67908418>:
>
> > @@ -71,6 +71,7 @@ public:
> >    G4String GetDetectorName()      {return WCDetectorName;}
> >    G4double GetWaterTubeLength()   {return WCLength;}
> >    G4double GetWaterTubePosition() {return WCPosition;}
> > +  G4double GetWaterTubeRadius()   {return WCRadius;}
>
> @federiconova <https://github.com/federiconova> Have you looked at 
> |WCSimDetectorConstruction::GetGeo_Dm(G4int i)| 
> <https://github.com/WCSim/WCSim/blob/2f7ca11b89dc893f115932186184a94614e43ac2/src/WCSimConstructGeometryTables.cc#L350>? 
> |GetGeo_Dm(3)| should give you the radius. (See e.g. this example 
> usage 
> <https://github.com/WCSim/WCSim/blob/2f7ca11b89dc893f115932186184a94614e43ac2/src/WCSimRunAction.cc#L142>.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/WCSim/WCSim/pull/168/files/6c154164188cb77bb4590da21bfcd48133f88281#r67908418>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANMXw0P8w3pUV2Z4vidLORMkE7gbdgc9ks5qOBiBgaJpZM4IutXq>.
>

-- 

_____________________________________________________________
Federico Nova         office: R1 2.89
phone: 01235 44 6980
email:federico.nova@stfc.ac.uk
_____________________________________________________________
Particle Physics Department            Rutherford Appleton Laboratory
Harwell, Didcot, Oxford, OX11 0QX, UK
_____________________________________________________________